libs/community/langchain_community/graphs/falkordb_graph.py

```diff
@@ -199,3 +199,19 @@ def add_graph_documents(
                     ),
                     {"properties": rel.properties},
                 )
+                
+    def switch_graph(self, graph_name: str) -> None:
+        """Switch to the given graph name (`graph_name`).
+        
+        This method allows users to change the active graph within the same 
+        database connection.
+
+        Args:
+            graph_name (str): The name of the graph to switch to.
+        """
+        self._graph = self._driver.select_graph(graph_name)
+        
+        try:
+            self.refresh_schema()
+        except Exception as e:
+            raise ValueError(f"Could not refresh schema. Error: {e}")
```

Comment:
Are there any additional state variables that need to change when switching graphs ?

Reply:
No